packages/feedback/src/integration.ts

```diff
@@ -0,0 +1,460 @@
+        light: Object.assign({}, DEFAULT_THEME.light, theme && theme.light),
+      },
+
+      buttonLabel,
+      cancelButtonLabel,
+      submitButtonLabel,
+      formTitle,
+      emailLabel,
+      emailPlaceholder,
+      messageLabel,
+      messagePlaceholder,
+      nameLabel,
+      namePlaceholder,
+      successMessageText,
+
+      onActorClick,
+      onDialogOpen,
+      onSubmitError,
+      onSubmitSuccess,
+    };
+
+    // TOOD: temp for testing;
+    this.setupOnce();
+  }
+
+  /** If replay has already been initialized */
+  /**
+   * Setup and initialize replay container
+   */
+  public setupOnce(): void {
+    if (!isBrowser()) {
+      return;
+    }
+
+    try {
+      // TODO: This is only here for hot reloading
+      if (this._host) {
+        this.remove();
+      }
+      const existingFeedback = document.querySelector(`#${this.options.id}`);
+      if (existingFeedback) {
+        existingFeedback.remove();
+      }
+      // TODO: End hotloading
+
+      const { attachTo, autoInject } = this.options;
+      if (!attachTo && !autoInject) {
+        // Nothing to do here
+        return;
+      }
+
+      // Setup host element + shadow DOM, if necessary
+      this._shadow = this._createShadowHost();
+
+      // If `attachTo` is defined, then attach click handler to it
+      if (attachTo) {
+        const actorTarget =
+          typeof attachTo === 'string'
+            ? document.querySelector(attachTo)
+            : typeof attachTo === 'function'
+            ? attachTo
+            : null;
+
+        if (!actorTarget) {
+          logger.warn(`[Feedback] Unable to find element with selector ${actorTarget}`);
+          return;
+        }
+
+        actorTarget.addEventListener('click', this._handleActorClick);
+      } else {
+        this._createWidgetActor();
+      }
+
+      if (!this._host) {
+        logger.warn('[Feedback] Unable to create host element');
+        return;
+      }
+
+      document.body.appendChild(this._host);
+    } catch (err) {
+      // TODO: error handling
+      console.error(err);
+    }
+  }
+
+  /**
+   * Removes the Feedback widget
+   */
+  public remove(): void {
+    if (this._host) {
+      this._host.remove();
+    }
+  }
+
+  /**
+   * Opens the Feedback dialog form
+   */
+  public openDialog(): void {
+    try {
+      if (this._dialog) {
+        this._dialog.open();
+        this._isDialogOpen = true;
+        if (this.options.onDialogOpen) {
+          this.options.onDialogOpen();
+        }
+        return;
+      }
+
+      try {
+        this._shadow = this._createShadowHost();
+      } catch {
+        return;
+      }
+
+      // Lazy-load until dialog is opened and only inject styles once
+      if (!this._hasDialogEverOpened) {
+        this._shadow.appendChild(createDialogStyles(document));
+      }
+
+      const userKey = this.options.useSentryUser;
+      const scope = getCurrentHub().getScope();
+      const user = scope && scope.getUser();
```

Comment:
I think `scope` should always be defined, we changed this some time ago, so we can drop the `scope &&` check here.